main_client.py

```diff
@@ -76,4 +76,12 @@ def main(username, password):
     #client.DownloadFile('test1.txt')
 
 
-    """
+"""
+
```

Comment:
why is this function commented?
if needed for later development, then delete the unnecessary blank spaces